### DISCUSSION TITLE
RTL (Arabic) is not supported by default in maplibre/mapbox vector tiles/styles basemaps

### DISCUSSION THREAD

**Discussion by yharby** [directus/directus#24931] [Feature Requests]
### Describe the Bug

When I try to add basemap as (mapbox style URL/JSON) in Arabic language, the Arabic letters are mirrored (displayed but not in correct way)

I think it's a JS plugin I must import in the code but for now, I want this to be in an official/custom docker image to run many microservices with the fix
The maplibre plugin to support RTL:
https://maplibre.org/maplibre-gl-js-docs/example/mapbox-gl-rtl-text/

I will try to fix and PR. 
Thank You

### To Reproduce

Add any custom basemap URL (not Raster for sure) with Arabic as a default language or any RTL language

### Errors Shown

![image](https://user-images.githubusercontent.com/13020265/189437914-1df0733e-3ca3-4652-8114-e706dcc65afe.png)

### What version of Directus are you using?

9.7.4

### What version of Node.js are you using?

16.16.0

### What database are you using?

PostgreSQL 14

### What browser are you using?

Firefox

### How are you deploying Directus?

Docker and Locally

**Comment by rijkvanzanten** [comment]
Linear: [ENG-192](https://linear.app/directus/issue/ENG-192/rtl-arabic-is-not-supported-by-default-in-maplibremapbox-vector)
